confgenerator/config.go

```diff
@@ -460,6 +498,24 @@ var (
 		"hostmetrics": collectionIntervalValidation,
 		"iis":         collectionIntervalValidation,
 		"mssql":       collectionIntervalValidation,
+		"exclude_metrics": map[string]func(interface{}) error{
+			"metric_prefixes": func(v interface{}) error {
+				var errors []string
+				for _, prefix := range v.([]string) {
+					if !strings.HasSuffix(prefix, "/*") {
+						errors = append(errors, fmt.Sprintf(`%q must end with "/*"`, prefix))
+					}
+					// TODO: Relax prefix must be `agent.googleapis.com/` check, when it starts supporting metrics with other prefixes
```

Comment:
Same question: what's "it"? And same nit: missing trailing `.`. So maybe something like this?

``// TODO: Relax the prefix check when we support metrics with other prefixes.``